src/components/AllJokes/AllJokesContainer.js

```diff
@@ -8,6 +13,58 @@ import { JokesList } from "./";
 import { Form, Button } from "react-bootstrap";
 
 const AllJoke = () => {
+  const DEFAULT_JOKES_AMMOUNT = 20;
+  const dispatch = useDispatch();
+  const allJokes = useSelector((state) => state.jokes);
+  const apiError = useSelector((state) => state.error);
+  const reduxJokesFilter = useSelector((state) => state.filter);
+  const [jokesFilter, setJokesFilter] = useState(reduxJokesFilter);
+  const [jokesAmmountToFetch, setJokesAmmountToFetch] = useState("");
+
+  function initRequestFetchJokes() {
+    if (!allJokes.length) {
+      dispatch(fetchJokesRequested(DEFAULT_JOKES_AMMOUNT));
```

Comment:
fetching should be implemented via sagas

Reply:
Isn't it? I don't get it. I mean, I'm dispatching the action and saga watches it. Then it makes fetching.